<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Page1">
     <property name="accessibleName">
      <string>Page1</string>
     </property>
     <property name="accessibleDescription">
      <string>Page1</string>
     </property>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QListWidget" name="listImages">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>531</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblMainImage">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>181</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="listFaces">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblFaceImage">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>240</y>
        <width>181</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImgFaceName">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>440</y>
        <width>531</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>420</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Face Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFindImages">
      <property name="geometry">
       <rect>
        <x>546</x>
        <y>10</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Images</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveName">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>440</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Assign Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page2">
     <property name="accessibleName">
      <string>Page2</string>
     </property>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
